<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DReports</class>
 <widget class="QDialog" name="DReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/resource/Reports.png</normaloff>:/new/resource/Reports.png</iconset>
  </property>
  <widget class="QGroupBox" name="optionsgroupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>521</width>
     <height>85</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
   <property name="title">
    <string>Search Options</string>
   </property>
   <widget class="QRadioButton" name="dateradio">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>20</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/resource/Searchcalender.png</normaloff>:/new/resource/Searchcalender.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="vehicleradio">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>20</y>
      <width>108</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Vehicle No</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/resource/Secondweight.png</normaloff>:/new/resource/Secondweight.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="materialradio">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Material</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/resource/Materialsearch.png</normaloff>:/new/resource/Materialsearch.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="receiptradio">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>108</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Receipt No</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/resource/Receiptsearch.png</normaloff>:/new/resource/Receiptsearch.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="customerradio">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>20</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Customer</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/resource/Customersearch.png</normaloff>:/new/resource/Customersearch.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="searchbutton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Icon/search1.png</normaloff>../../Icon/search1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>23</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>51</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_edit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>51</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="month_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>51</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="year_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>51</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="search_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>51</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reciept No : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="seperator_label1">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>51</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="seperator_label2">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="previousbutton">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>408</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextbutton">
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>408</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="displayedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>521</width>
     <height>310</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="printbutton">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>408</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/resource/Print.png</normaloff>:/new/resource/Print.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="clearbutton">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>408</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>receiptradio</tabstop>
  <tabstop>vehicleradio</tabstop>
  <tabstop>customerradio</tabstop>
  <tabstop>materialradio</tabstop>
  <tabstop>dateradio</tabstop>
  <tabstop>date_edit</tabstop>
  <tabstop>search_edit</tabstop>
  <tabstop>month_edit</tabstop>
  <tabstop>year_edit</tabstop>
  <tabstop>searchbutton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
